core/src/main/java/org/apache/accumulo/core/classloader/URLContextClassLoaderFactory.java

```diff
@@ -63,19 +62,15 @@ public ClassLoader getClassLoader(String context) {
       throw new IllegalArgumentException("Unknown context");
     }
 
-    try {
-      return classloaders.get(context, () -> {
-        LOG.debug("Creating URLClassLoader for context, uris: {}", context);
-        return new URLClassLoader(Arrays.stream(context.split(",")).map(url -> {
-          try {
-            return new URL(url);
-          } catch (MalformedURLException e) {
-            throw new RuntimeException(e);
-          }
-        }).collect(Collectors.toList()).toArray(new URL[] {}), ClassLoader.getSystemClassLoader());
-      });
-    } catch (ExecutionException e) {
-      throw new RuntimeException(e);
-    }
+    return classloaders.get(context, k -> {
+      LOG.debug("Creating URLClassLoader for context, uris: {}", context);
+      return new URLClassLoader(Arrays.stream(context.split(",")).map(url -> {
+        try {
+          return new URL(url);
+        } catch (MalformedURLException e) {
+          throw new RuntimeException(e);
+        }
+      }).collect(Collectors.toList()).toArray(new URL[] {}), ClassLoader.getSystemClassLoader());
```

Comment:
```suggestion
      }).toArray(URL[]::new), ClassLoader.getSystemClassLoader());
```
Outside of the scope and not changed in this PR but noticed that the collect step can be skipped here.

Reply:
Good catch, I will go ahead and fix that.